scripts/us_epa/air_emissions_inventory/county/metadata.py

```diff
@@ -0,0 +1,26 @@
+source_pollutant = {
```

Comment:
nit: make these constants capital. Here and everywhere else in the PR

Reply:
made the constants capital